Terran-One/cw-simulate-ui
# Replace diff switch with cancel sign.

## Issue link

[WL-496](https://terran-one.atlassian.net/browse/WL-496)

## Description
Replaced diff switch with cross button.

## Test steps

1. Go to simulation page ( after instantiation)
2. Compare States.
3. Now you can see cross button instead of diff switch. (Click on it, it will exit state diff)
